ugvc/cnv/create_reads_count_cohort_matrix.R

```diff
@@ -0,0 +1,50 @@
+suppressPackageStartupMessages(library(cn.mops))
```

Comment:
Can you please add the license header to this script?

Reply:
Do you think we need to add the license header to the following scripts as well? : 
ugvc/cnv/get_reads_count_from_bam.R
ugvc/cnv/merge_reads_count_sample_to_cohort.R